final_prep/annotations.rb

```diff
@@ -4,40 +4,66 @@
 
 # Build a Bear
 
+# define a method named build_a_bear passing in 5 arguments
```

Comment:
A better way to say this would be "declaring 5 _parameters_" (not arguments). Arguments are when you pass in the actual values. Parameters and arguments are 2 sides of the same coin, so to speak